images/daemon/image/daemon.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: apps/v1 
+kind: DaemonSet
+metadata:
+  name: kata-install-daemon
+  #namespace: kata-operator
+spec:
+  selector:
+    matchLabels:
+      name: kata-install-daemon 
+  template:
+    metadata:
+      labels:
+        name: kata-install-daemon 
+    spec:
+      serviceAccountName: kata-operator
+      containers:
+      - name: kata-install-daemon 
+        image: quay.io/harpatil/kata-install-daemon:1.5
```

Comment:
why is it referring to my personal image :)

Reply:
not sure, somebody left it there ;-)

Reply:
before I merge this I'll push the new image to quay.io/isolatedcontainers/kata-operator-daemon:4.7 and test it